bertopic/_bertopic.py

```diff
@@ -256,22 +256,41 @@ def __init__(self,
         self.topic_mapper_ = None
         self.topic_representations_ = None
         self.topic_embeddings_ = None
-        self.topic_labels_ = None
+        self.topic_id_to_zeroshot_topic_idx = {}
```

Comment:
I'm using the `_` at the end of the variable to distinguish between parameters/public attributes/private attributes. Could you update this so it is inline with the structure of the public attributes? Unless you believe this variable should not be accessed by the user at all, then we can convert it to a private attribute.

Reply:
You're right. Forgot about this. Made it private.